doc/main/reference/blocking_terminate.rst

```diff
@@ -60,16 +62,43 @@ Synopsis
 Member Functions
 ----------------
 
+.. cpp:function:: task_scheduler_handle()
+
+    **Effects**: Creates an instance of the ``task_scheduler_handle`` class that does not contain any reference to a task scheduler
+
+-------------------------------------------------------
+
 .. cpp:function:: ~task_scheduler_handle()
 
     **Effects**: Destroys an instance of the ``task_scheduler_handle`` class.
     Releases a reference to the task scheduler and deactivates an instance of the ``task_scheduler_handle`` class.
 
 -------------------------------------------------------
 
+.. cpp:function:: task_scheduler_handle(task_scheduler_handle&& other) noexcept
+
+    **Effects**: Creates an instance of the ``task_scheduler_handle`` class that references to the task scheduler referenced with ``other``.
+    ``other`` releases a reference to the task scheduler.
+
+-------------------------------------------------------
+
+.. cpp:function:: task_scheduler_handle& operator=(task_scheduler_handle&& other) noexcept
+
+    **Effects**: Releases a reference to the task scheduler referenced with ``this`` and adds a reference to the task scheduler referenced with ``other``.
+    ``other`` releases a reference to the task scheduler.
+
+-------------------------------------------------------
+
+.. cpp:function:: explicit operator bool() const noexcept
+
+    **Returns**: ``true`` if ``this`` references any task scheduler; otherwise, returns ``false``
+
+-------------------------------------------------------
+
 .. cpp:function:: task_scheduler_handle get()
 
-    **Returns**: An instance of the ``task_scheduler_handle`` class.
+    **Returns**: An instance of the ``task_scheduler_handle`` class that holds a references to the task scheduler preventing
```

Comment:
```suggestion
    **Returns**: An instance of the ``task_scheduler_handle`` class that holds a reference to the task scheduler preventing
```

Reply:
Any notes that it should be called before TBB initialization?